pkg/terraform/exec/plugins/Gopkg.lock

```diff
@@ -795,6 +795,14 @@
   revision = "9f5bba54dad78c10ed572eb2541f8178a5500656"
   version = "v0.1.4"
 
+[[projects]]
+  digest = "1:a0886267c60262a245d6eeb0be12bf8e165026ad088bce106b9c55c5d613144f"
+  name = "github.com/openshift-metalkube/terraform-provider-ironic"
```

Comment:
Ooh sorry looks like this got included too, we should only use the metal3 version, I'd remove this from Gopkg.lock and the openshift-metalkube directory in vendor